### DISCUSSION TITLE
Error when packaging SOFA Unity, physics disappear

### DISCUSSION THREAD

**Discussion by 985-211** [InfinyTech3D/SofaUnity#75] [Help - Using SofaUnity]
When importing SOFA into Unity, physics effects are present, but after packaging, why do all the physics effects disappear?

**Comment by epernod** [comment]
Hi,

there might be a problem of license path or sofa.ini paths. 
Could you specify the type of build you are doing and provide the player.log
It should be located somewhere like: %USERPROFILE%\AppData\LocalLow\CompanyName\ProductName\Player.log
see https://docs.unity3d.com/Manual/LogFiles.html

Best regards

**Reply by 985-211** [reply]
[Player.log](https://github.com/InfinyTech3D/SofaUnity/files/14953493/Player.log) This is my log, please help me analyze what caused the disappearance of the sofa's physical effects after packaging. I look forward to your reply

**Reply by 985-211** [reply]
Please Help Me!

**Reply by epernod** [reply]
Hi,

according to the player log, you have an error of path to the scene file:
```
SofaContextAPI::loadScene method returns: -18: Simulation scene file (.scn) not found. for scene: D:/UnityHub/UnityProject/HdrpBreast/game/HdrpBreast_Data/Test/Test(1).scn
SofaDAGNodeManager Error while reconnecting the graph: 8 DAGNode found in Unity instead of : 1 from SOFA side.
```

Could you check that this file exist? Then could you show what is your project architecture before building.
Best regards,

**Reply by 985-211** [reply]
Thank you very much, I have already solved the problem

**Reply by epernod** [reply]
Hi,
good news, could you just post your solution here to keep track for next users.
Thanks,

**Comment by epernod** [comment]
Hi,

as I didn't get any response from you, I'm closing this discussion thread. Feel free to reopen it or ask other questions in another thread.

Best regards,
